src/style.css

```diff
@@ -7,7 +7,7 @@
 /* This adds some low-level css resets based on the browserslist in package.json, see https://create-react-app.dev/docs/adding-css-reset */
 @import-normalize;
 
-.hover-style:hover {
+.hover-style:hover, .hover-style:focus {
```

Comment:
These two changes ensure that the hover styles work when the user moves their focus to the element using the tab key, not just the mouse. Since the focus outlines are not always easy to see in Terra UI, it's rather jarring if the hover states don't appear.